tests/test_tutorial/test_django_orm/test_tutorial001.py

```diff
@@ -0,0 +1,68 @@
+import pathlib
+import sys
+
+import pytest
+from django.core.management.color import no_style
+from django.core.management.sql import sql_flush
+from django.db import connection
+from django.utils import timezone
+from fastapi.testclient import TestClient
+
+HERE = pathlib.Path(__file__).parent
+
+sys.path.append(str(HERE.parents[2] / "docs_src" / "django_orm"))
```

Comment:
I'm usually wary of messing with `sys.path`, is this the only way to make this work?